build.sbt

```diff
@@ -1,12 +1,33 @@
-lazy val testSuite = (project in file("."))
-  .enablePlugins(SbtAutoBuildPlugin)
-  .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
-  .settings(
-    name := "debt-transformation-acceptance-tests",
-    version := "0.1.0",
-    scalaVersion := "2.12.12",
-    scalacOptions ++= Seq("-feature"),
-    libraryDependencies ++= Dependencies.test,
-    //The testOptions from SbtAutoBuildPlugin supports only ScalaTest. Resetting testOptions for Cucumber Tests.
-    testOptions in Test := Seq.empty
-  )
+name := "debt-transformation-acceptance-tests"
+version := "0.1"
+scalaVersion := "2.12.11"
+
+val CucumberVersion = "4.8.0"
+
+resolvers += "hmrc-releases" at "https://artefacts.tax.service.gov.uk/artifactory/hmrc-releases/"
+
+libraryDependencies ++= Seq(
+  "com.typesafe.play"          %% "play-ahc-ws-standalone"  % "1.1.7",
+  "com.typesafe.play"          %% "play-ws-standalone-json" % "1.1.7",
+  "com.typesafe.play"          %% "play-ws-standalone-xml"  % "1.1.7",
+  "io.github.bonigarcia"       % "webdrivermanager"         % "3.7.1",
```

Comment:
Can you remove this dependency as we don't have any UI to automate?

Reply:
removed